native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js

```diff
@@ -16,18 +16,25 @@
  * limitations under the License.
  */
 
+import BpkText from './BpkText';
 import commonTests from './BpkText-test.common';
 
 jest.mock('react-native', () => {
   const reactNative = require.requireActual('react-native');
   jest
-      .spyOn(reactNative.Platform, 'select')
-      .mockImplementation(obj => obj.ios || obj.default);
+    .spyOn(reactNative.Platform, 'select')
```

Comment:
why remove these? :P

Reply:
We really need to integrate prettier to eradicate these sorts of stylistic nitpicks.

Reply:
I've got the auto-formatter on so it got removed automatically to comply with the settings in root of the project (I hope)